sympy/solvers/pde.py

```diff
@@ -662,26 +677,26 @@ def pde_1st_linear_constant_coeff(eq, func, order, match, solvefun):
 def pde_1st_linear_variable_coeff(eq, func, order, match, solvefun):
     r"""
     Solves a first order linear partial differential equation
-    with variable coefficients. The general form of this partial differential equation is
+    with variable coefficients. The general form of this partial
+    differential equation is
 
     .. math:: a(x, y) \frac{df(x, y)}{dx} + a(x, y) \frac{df(x, y)}{dy}
-                + c(x, y) f(x, y) - G(x, y)
+                + c(x, y) f(x, y) = G(x, y)
 
-    where `a(x, y)`, `b(x, y)`, `c(x, y)` and `G(x, y)` are arbitrary functions
-    in `x` and `y`. This PDE is converted into an ODE by making the following transformation.
+    where `a(x, y)`, `b(x, y)`, `c(x, y)` and `G(x, y)` are arbitrary
+    functions in `x` and `y`. This PDE is converted into an ODE by
+    making the following transformation.
 
-    1] `\xi` as `x`
+    1. `\xi` as `x`
 
-    2] `\eta` as the constant in the solution to the differential equation
-    `\frac{dy}{dx} = -\frac{b}{a}`
+    2. `\eta` as the constant in the solution to the differential
+       equation `\frac{dy}{dx} = -\frac{b}{a}`
 
     Making the following substitutions reduces it to the linear ODE
 
     .. math:: a(\xi, \eta)\frac{du}{d\xi} + c(\xi, \eta)u - d(\xi, \eta) = 0
```

Comment:
I think `d(xi, eta)` had come from nowhere

<img src="https://user-images.githubusercontent.com/34944973/53187399-6153c000-3646-11e9-82b7-f565d6cb9870.png" width="256px">

But after reading Viktor Grigoryan's reference, I think it is naively denoting `G(xi, eta)`,
So either this one can be changed to `G(xi, eta)`, or the equation above can be changed to `d(xi, eta)`.

And also, the example section should align with the new formula

Reply:
Done.